### ISSUE TITLE
Treegrid : JAWS doesn’t announce the column and row count for the “treegrid” role on initial focus

### ISSUE THREAD

**Issue by havvamehfooz** [FreedomScientific/VFO-standards-support#930]
Summary

JAWS doesn’t announce the column and row count for the “treegrid” role on initial focus.
In this particular sample below, the expected announcement would be: “treegrid with 2 of 5 columns and 2 of 34 rows”. However, JAWS only announces the role. Interestingly, when navigated to the treegrid in VPC Mode the count information is announced as expected, but not on focus.

Plain html [example] : https://snippix.only.sap/#986007

JAWS version: Jaws 2025

**Comment by JeremiahRogersVispero** [comment]
@havvamehfooz , please check the URL in this post and revise. Thank you.

**Comment by havvamehfooz** [comment]
@JeremiahRogersVispero : I checked the URL and it works fine

![Image](https://github.com/user-attachments/assets/f344caa5-a787-49fc-a4fc-21fa20e7a941)

**Comment by JeremiahRogersVispero** [comment]
> [@JeremiahRogersVispero](https://github.com/JeremiahRogersVispero) : I checked the URL and it works fine
> 
> ![Image](https://github.com/user-attachments/assets/f344caa5-a787-49fc-a4fc-21fa20e7a941)

Unfortunately, two colleagues and I have all tried to access the URL and could not. Can you please share via a different means or send along the code via a different means? Thanks in advance.

**Comment by havvamehfooz** [comment]
@JeremiahRogersVispero : Could you please try the below link?
https://jsbin.com/pezesanizo/edit?html,output
